tests/functional/Vagrantfile

```diff
@@ -0,0 +1,145 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+NMONS = 3
+NOSDS = 3
+NMDSS = 0
+NRGWS = 0
+NNFSS = 0
+MGRS  = 0
```

Comment:
Eventually would probably make sense to not differentiate between types of hosts and only have a single type of host. It's a copy of https://github.com/ceph/ceph-ansible/blob/83a8dd5a6a1f9ffe43b2a75d7b49775e34c58f24/Vagrantfile right? I think we can simplify this later on.

Reply:
@sebastian-philipp this file is for CI usage only, so yes we can update it later if needed

Reply:
@sebastian-philipp, it might make sense to differentiate between types of hosts but this is another topic that has nothing to do here, I guess we should discuss it when the time comes later if we decide to have more automation on top of cephadm.

Reply:
What is the goal of splitting services onto multiple hosts in this case? Work is already being done to test that services may be deployed to individual instances by Ceph, so this seems like a duplication of effort. This playbook would likely be validated by using 3 static instances since all non-playbook reasons for that failing would be upstream bugs.